kubernetes/README.adoc

```diff
@@ -0,0 +1,301 @@
+= Kubernetes
+Snowdrop Team (Antonio costa)
+:icons: font
+:icon-set: fas
+:revdate: {docdate}
+:toc: left
+:toclevels: 3
+:description: This document describes the requirements, and the process to execute to install a k8s cluster on a host. The installation will be done using Ansible.
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+== Introduction
+
+This document describes the requirements, and the process to execute to
```

Comment:
i suppose that we more than one process ;-) Better is to use procedure

Reply:
Improved the kubernetes documentation.